frontend/src/components/eMIB/transformations.js

```diff
@@ -0,0 +1,28 @@
+// transform address book into list of options
```

Comment:
Options for what? Can you add to this to make this comment more specific? 
Ex - addressBook is of shape ___ and the function returns an array of ___

Reply:
added comments in 037bbbc